_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,84 @@
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/banfield_data_theory.svg" alt="banfield_data_theory" width="800px" style="padding:5px;">
+</p>
+
+Does this mean that the study that found high correlation between replicates was wrong to conclude that acquisition was biased? Not at all, in fact their study demonstrates that beyond a doubt! But here's the very important catch: our work shows that if you observe a broad distribution by itself, you *cannot* conclude that biased acquisition is the cause, because we know you don't need that ingredient to get a similar result. These things are all true at once: spacer acquisition IS biased, biased acquisition CAN lead to broad abundance distributions, BUT observing a broad abundance distribution does not guarantee that acquisition is biased. We have found evidence of at least two processes that can lead to broad spacer abundance distributions - biased acquisition and randomness - which means when we observe a broad distribution, we can't know right away which is the cause.
+
+Why does this matter? In lab experiments like those described here, you can do multiple copies of the same experiment under the same conditions to prove a result. But the world is full of important and fascinating research areas where that isn't possible. Any study of bacteria in a natural environment is inherently irreproducible: you can never turn back the clock and do things the same way twice. So you trek to the glacial meltwater lake, sequence millions of bacteria, find all their CRISPR spacers, and plot the abundance distribution. How can you figure out what's going on? In biology *everything* is going on, and the key to understanding is to figure out *which* of the things going on are important for what you observe and which are not. Our null model taught us that if you want to learn about spacer acquisition, you'll need to look beyond a broad spacer abundance distribution to find it. 
```

Comment:
What is the 'it' referred to at the end of the paragraph?

Reply:
Removed "to find it" - I think that makes it more clear.